test/avocado/selenium-sosreport.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+# we need to be able to find and import seleniumlib, so add this directory
+import os
+import sys
+import time
+machine_test_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(1, machine_test_dir)
```

Comment:
yep, it sounds good, but it is more less workaround, when I've talked to avocado devels, they said that now there is implemented that this path is appended to path, so if it will be in all distros, I can remove that.
But yep, I can use it, it also solve that situation when avocado devels add this path to actual paths